content/departments/cloud/working-agreements.md

```diff
@@ -86,3 +162,90 @@ Operational knowledge:
 - On-call playbooks, troubleshooting/operational documentation
 - Cloud team knowledge sharing focused on operator maintainence and future dev work (e.g. with docs or walkthroughs)
 - Launch questionnaire or playbook
+
+### Cloud Engineer
+
+- Make continuous progress on roadmap items
+- Implementation work
+- Push for architecture/feature designs that work the best for the cloud
+- Regularly team the team on their work, encountered problems, blockers, and delays
+- Highlight potential security issues for review
+- Complete work as defined by DRIs
+- Documentation and testing
+- Ensure the team stays technically flexible to deliver on changing requirements
+- Identify areas of technical improvement, process improvement, and paring down tech debt
+- Technical decision-making
+- Code reviews of Go code and IaaC languages like Terraform
+- Identify and escalate gaps in processes, tooling, and ownership that affect Cloud roadmap
+- Identify team weaknesses and define processes for improvement
+- Maintain a broad view on state and status of team initiatives and roadmap items
+- Grow and maintain knowledge of GCP resources and when to use managed resources vs self-hosting
+
+### Cloud EM and PM
+
+- Lead and represent the team to external stakeholders
+- Make sure the rest of company is aware of Cloud deliverables
+- Align the Cloud roadmap with external priorities
+- Support the DRIs with cross-team communication and establish requirements from other teams
+- Collect and share with with the team future goals and plans -- i.e. things to do for strategic customers -- to better plan the QX roadmap
+- Identify and escalate gaps in processes, tooling, and ownership that affect the Cloud roadmap
+- Maintain processes for how we work
+- Track progress of larger initiatives
+- Maintain a broad view on state and status of team initiatives and roadmap items
+- Manage Cloud dependencies on other teams
+- Manage relationship with cross-function peers (CE, Sales, Support, Legal etc)
+- Fight for headcount
```

Comment:
Could reframe this to something like `define and execute staffing strategy to meet the business expectations`